framework/wazuh/core/utils.py

```diff
@@ -791,18 +791,14 @@ def check_disabled_limits_in_conf(data):
     WazuhError(1127)
         Raised if one of the disabled limits is present in the configuration to upload.
     """
-    blocked_configurations = configuration.api_conf['upload_configuration']
-
-    xml_file = fromstring(f"<root_tag>{data}</root_tag>")
-    found_limits = []
-    for global_section in xml_file.findall("global"):
-        found_limits += [limit_section for limit_section in global_section.findall("limits") or []]
-    if len(found_limits) == 0:
-        return
+    def check_section(section_name):
+        if re.findall(pattern=r'<global>.*<limits>.*<{0}>.*</{0}>.*</limits>.*</global>'.format(section_name),
+                      string=data, flags=re.MULTILINE | re.DOTALL | re.IGNORECASE):
+            raise WazuhError(1127, extra_message=f"global > limits > {disabled_limit}")
```

Comment:
I think it should be `{section_name}` instead of `{disabled_limit}`.

Reply:
Fixed in https://github.com/wazuh/wazuh/pull/16455/commits/962ce760e1e3ccff949b50ebac8f1393abf09d67